test-project-app/package.json

```diff
@@ -6,10 +6,14 @@
   "author": "Paolo Caleffi <p.caleffi@dreamonkey.com>",
   "private": true,
   "scripts": {
-    "sync:testing": "cd ../packages/testing && yarn build:local && cd ../../test-project-app && yarn add -D ../packages/testing",
-    "sync:jest": "cd ../packages/unit-jest && yarn build:local && cd ../../test-project-app && yarn add -D ../packages/unit-jest",
-    "sync:cypress": "cd ../packages/e2e-cypress && yarn build:local && cd ../../test-project-app && yarn add -D ../packages/e2e-cypress",
+    "sync:testing": "yarn --cwd ../packages/testing build:local && yarn add -D ../packages/testing",
```

Comment:
I copied your way of doing this so all projects commands now work cross-os, thanks for the hint

Reply:
Yes I noticed, I found out about this a while back :)